docs/src/code/building-linuxcnc.adoc

```diff
@@ -54,24 +95,23 @@ $ source ../scripts/rip-environment
 $ runtests
 -----
 
-This might fail too!  Read this whole document, but especially the section
-on <<Setting-up-the-environment,Setting up the test environment>>.
+This might fail, too!
+Read this whole document, especially the section on <<Setting-up-the-environment,Setting up the test environment>>.
```

Comment:
I assume you are aware that these whitespace changes do not affect the translation file content and the only outside effect is adjusting the 'who change the string last' value in git.

Why do you still believe changes like this is needed?

Reply:
https://hosted.weblate.org/translate/linuxcnc/linuxcnc-docs/de/?q=state%3Aneeds-editing&sort_by=-priority%2Cposition&offset=33 is an example of what was now fixed.

Reply:
Oh, well.  I find it a waste of time only confusing 'git blame', but do not care much.

Reply:
I'd side with you if this was something functional like code. It is a bit more clumsy but no information is lost. Git log gives you the commit hash prior to the whitespace diting and you can then pass that to git blame.